docs/src/architecture/08_concepts/document_templates/proposal/F14-Generic-Steven/0ce8ab38-9258-4fbc-a62e-7faa6e58318f.schema.json

```diff
@@ -0,0 +1,274 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Catalyst Fund 14 Base Proposal Template",
+    "description": "A structured template for creating Fund 14 proposals",
+    "definitions": {

```

Comment:
JsonSchema has structuring tags one of which is $def, should that be used instead of "definitions" for the purpose of defining reusable structures
https://json-schema.org/understanding-json-schema/structuring#defs